frontend/src/metabase/new/components/NewModals/NewModals.tsx

```diff
@@ -91,10 +94,24 @@ export const NewModals = withRouter((props: WithRouterProps) => {
         </Modal>
       );
     case "embed": {
+      const props = currentNewModalProps as SdkIframeEmbedSetupModalProps;
       return (
-        <PLUGIN_EMBEDDING_IFRAME_SDK_SETUP.SdkIframeEmbedSetupModal
+        <SdkIframeEmbedSetupModal
           opened
-          initialState={currentNewModalProps?.initialState}
+          initialState={props?.initialState}
+          onClose={handleModalClose}
+        />
+      );
+    }
+    case STATIC_LEGACY_EMBEDDING_TYPE: {
+      const props = currentNewModalProps as LegacyStaticEmbeddingModalProps;
+
+      return (
+        <LegacyStaticEmbeddingModal
```

Comment:
How do you trigger this? I couldn't find a button to open the legacy static embedding modal